### PR TITLE
pdftools.pdfposter init at 0.7.post1

### PR DESCRIPTION
**Pull request by wamserma**
###### Motivation for this change

pdfposter is packaged for other distriutions and easy enough to package for NixOS so I can maintain it

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pdfposter/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  propagatedBuildInputs = [ pypdf2 ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0c1avpbr9q53yzq5ar2x485rmp9d0l3z27aham32bg7gplzd7w0j";
+  };
+
+  meta = with stdenv.lib; {
+    description = "pdfposter is much like poster does for Postscript files, but working with PDF.";
```
Comment (symphorien) [inline pkgs/development/python-modules/pdfposter/default.nix]: `meta.description` should:

*    Be capitalized.
*    Not start with the package name.
*    Not have a period at the end.
(see https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md)

**Review by symphorien — COMMENTED** [review]
I compiled and ran successfully the tool on a random pdf.

Apart from the stylistic comment above, this PR looks good.

File: pkgs/development/python-modules/pdfposter/default.nix (hunk 1 above)
Comment (wamserma) [reply pkgs/development/python-modules/pdfposter/default.nix]: Ha, managed to violate all three points with a single commit. Fixed now.

**Review by symphorien — APPROVED** [review]

**Comment by wamserma** [comment]
@FRidh @jonringer could you please have a look?

File: pkgs/development/python-modules/pdfposter/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    homepage = https://gitlab.com/pdftools/pdfposter;
```
Comment (veprbl) [inline pkgs/development/python-modules/pdfposter/default.nix:16]:
```suggestion
    homepage = "https://pdfposter.readthedocs.io";
```

File: pkgs/development/python-modules/pdfposter/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ wamserma ];
+    platforms = platforms.all;
```
Comment (veprbl) [inline pkgs/development/python-modules/pdfposter/default.nix:19]: Please remove this line – this is already set by `buildPythonPackage`

File: pkgs/development/python-modules/pdfposter/default.nix (hunk 2 above)
Comment (wamserma) [reply pkgs/development/python-modules/pdfposter/default.nix]: Fixed.

File: pkgs/development/python-modules/pdfposter/default.nix (hunk 3 above)
Comment (wamserma) [reply pkgs/development/python-modules/pdfposter/default.nix]: Fixed.

**Comment by veprbl** [comment]
@GrahamcOfBorg build python27Packages.pdfposter
@GrahamcOfBorg build python37Packages.pdfposter
@GrahamcOfBorg build python38Packages.pdfposter

**Review by veprbl — APPROVED** [review]

**Pull request merged**